src/main/io/serial_cli.c

```diff
@@ -1410,6 +1410,7 @@ static void cliReboot(void) {
     cliPrint("\r\nRebooting");
     waitForSerialPortToFinishTransmitting(cliPort);
     stopMotors();
+    handleOneshotFeatureChangeOnRestart();
```

Comment:
Thanks. Yes, i am aware of that. There is (was) already a delay to do this in the stopMotors()  function itself.
It even has the same comment :smile: 
```C
void stopMotors(void)
{
    writeAllMotors(escAndServoConfig->mincommand);

    delay(50); // give the timers and ESCs a chance to react.
}
```
In the original situation the soft reset came directly after that function call, so that meant killing the PWM output with the same timing as it is now